E2E/tests/teamList.test.ts

```diff
@@ -0,0 +1,106 @@
+Feature('team list');
+
+import username = require("../test_data/usernames");
+import ChessTile = require("../test_data/chessTiles");
+import { ChessPiece } from "../test_data/chessPieces";
+
+const { I, login, game } = inject();
+
+Scenario('Players who skipped their moves are at the bottom of the list', async () => {
+    login.firstVoterLogin(username.user1);
+    let url = await I.grabCurrentUrl();
+    session(username.user2, () => {
+        login.voterLoginIntoCreatedGameRoom(url, username.user2);
+    });
+    session(username.user3, () => {
+        login.voterLoginIntoCreatedGameRoom(url, username.user3);
+    });
+    session(username.user4, () => {
+        login.voterLoginIntoCreatedGameRoom(url, username.user4);
+        game.vote(ChessPiece.queen, ChessTile.f6);
+    });
+    session(username.user2, () => {
+        game.vote(ChessPiece.bishop, ChessTile.a1);
+    });
+    session(username.user3, () => {
+        I.click(game.locator.buttons.skip);
+        I.click(game.locator.buttons.skipOtherPlayer(username.user1));
+        I.waitForText('Game complete');
+        const expectedUserOrder = [
+            username.user4,
+            username.user2,
+            username.user3,
+            username.user1,
+        ];
+        expectedUserOrder.forEach((username, index) => {
+            I.see(`${username}`, game.locator.playersList.playerUsernameByIndex(index+=1));
```

Comment:
`index += 1` increments `index` value by one and assigns the new value to `index`, its a shorthand for
```
index = index + 1
```

I don't think we need it here, `index + 1` should be enough